src/main/java/org/sagebionetworks/web/client/widget/entity/registration/WidgetRegistrarImpl.java

```diff
@@ -61,6 +61,8 @@ public WidgetEditorPresenter getWidgetEditorForWidgetDescriptor(WikiPageKey wiki
 			presenter = ginInjector.getProvenanceConfigEditor();
 		} else if (contentTypeKey.equals(WidgetConstants.IMAGE_CONTENT_TYPE)) {
 			presenter = ginInjector.getImageConfigEditor();
+		} else if (contentTypeKey.equals(WidgetConstants.EXTERNAL_IMAGE_CONTENT_TYPE)) {
+			presenter = ginInjector.getExternalImageConfigEditor();
```

Comment:
Will probably also want to call registerWidget in initWithKnownWidgets() for this new content type (see the dialog title in the new editor).